src/index.html

```diff
@@ -16,7 +16,19 @@
       href="./styles/index.scss"
```

Comment:
The `href` attribute of your `link` tag is pointing to an `.scss` file. However, browsers can't interpret SCSS, they need a compiled CSS file. You should compile your SCSS to CSS and then link the CSS file in your HTML.